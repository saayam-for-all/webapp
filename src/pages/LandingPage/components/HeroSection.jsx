import React from "react";

const HeroSection = () => {
  return (
    <div className="hero bg-base-100">
      <div className="hero-content text-center">
        <div className="max-w-md">
          <div className="py-6 text-2xl">
            Welcome to <strong>Saayam</strong>, a non-profit organization â€“ is
            your gateway to assistance and support in times of need. Saayam
            means Help in Telugu, Sahaay in Sanskrit/Hindi, Bangzhu in Mandarin,
            and Ayuda in Spanish.
          </div>
        </div>
      </div>
    </div>
  );
};

export default HeroSection;
