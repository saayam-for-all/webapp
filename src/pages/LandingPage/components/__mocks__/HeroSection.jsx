export default function HeroSection() {
  return <mock-hero-setion />;
}
