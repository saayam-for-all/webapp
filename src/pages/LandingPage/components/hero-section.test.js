import { render } from "@testing-library/react";
import HeroSection from "./HeroSection";

describe("HeroSection", () => {
  it("renders correctly", () => {
    const tree = render(<HeroSection />);
    expect(tree).toMatchSnapshot();
  });
});
