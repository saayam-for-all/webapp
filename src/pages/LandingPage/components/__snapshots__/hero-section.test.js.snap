// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroSection renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="hero bg-base-100"
      >
        <div
          class="hero-content text-center"
        >
          <div
            class="max-w-md"
          >
            <div
              class="py-6 text-2xl"
            >
              <mock-trans
                props="{"i18nKey":"HERO_TEXT"}"
              >
                Welcome to 
                <strong>
                  Saayam
                </strong>
                , a non-profit organization – is your gateway to assistance and support in times of need. Saayam means Help in Telugu, Sahaay in Sanskrit/Hindi, Bangzhu in Mandarin, and Ayuda in Spanish.
              </mock-trans>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="hero bg-base-100"
    >
      <div
        class="hero-content text-center"
      >
        <div
          class="max-w-md"
        >
          <div
            class="py-6 text-2xl"
          >
            <mock-trans
              props="{"i18nKey":"HERO_TEXT"}"
            >
              Welcome to 
              <strong>
                Saayam
              </strong>
              , a non-profit organization – is your gateway to assistance and support in times of need. Saayam means Help in Telugu, Sahaay in Sanskrit/Hindi, Bangzhu in Mandarin, and Ayuda in Spanish.
            </mock-trans>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
